examples/hosts/app-integration/schema-upgrade/README.md

```diff
@@ -0,0 +1,40 @@
+# @fluid-example/app-integration-external-data
```

Comment:
Here and line 21, need to update to the correct package name (I think repo-policy-check is supposed to fail if the readme title doesn't match the package name?)